src/nyc_trees/nyc_trees/settings/production.py

```diff
@@ -32,11 +33,7 @@ def get_env_setting(setting):
 # ELBs use the instance IP in the Host header and ALLOWED_HOSTS checks against
 # the Host header.
 try:
-    ALLOWED_HOSTS.append(
-        urlopen(
-            'http://instance-data.ec2.internal/latest/meta-data/local-ipv4'
-        ).readline()
-    )
+    ALLOWED_HOSTS.append(get_instance_metadata()['local-ipv4'])
 except URLError:
```

Comment:
It looks like the underlying function here traps the `URLError` and returns an [empty string](https://github.com/boto/boto/blob/44d873d97bb08a09832b21dcaa0b2d47fe59411b/boto/utils.py#L198-L230).